smithy-model/src/test/java/software/amazon/smithy/model/loader/InvalidSmithyModelLoaderRunnerTest.java

```diff
@@ -67,15 +67,15 @@ private String cleanErrorMessage(String errorMessage) {
                 .replace("[EOF]", "")
                 // Make sure the line separators and representations of them are consistent across
                 // operating systems.
+                .replace("\n\n", "\\n")
```

Comment:
Why normalized two newlines into one?

Reply:
Windows and Linux behavior is divergent on an error string produced. When matching the entire error Windows saw two '\n' in a row and the other didn't, or the opposite. This was how I addressed it.

https://github.com/awslabs/smithy/pull/1213#discussion_r931652635

I changed the test case itself to match substring, however, the issue is certainly still there. Windows and Linux are not behaving the same.